srtcore/channel.cpp

```diff
@@ -162,7 +162,7 @@ void CChannel::open(const sockaddr* addr)
       if (0 != ::getaddrinfo(NULL, "0", &hints, &res))
          throw CUDTException(MJ_SETUP, MN_NORES, NET_ERROR);
 
-      if (0 != ::bind(m_iSocket, res->ai_addr, (int) res->ai_addrlen))
+      if (0 != ::bind(m_iSocket, res->ai_addr, (socklen_t) res->ai_addrlen))
```

Comment:
This cast is useless - `ai_addrlen` field **is** of type `socklen_t`.

Reply:
Ok, then looxlike this cast is then the only warning-proof solution. Add at least some short comment that explains this cast.